packages/lb-annotation/src/core/toolOperation/basicToolOperation.ts

```diff
@@ -220,7 +220,7 @@ class BasicToolOperation extends EventListener {
     this.defaultAttribute = props?.defaultAttribute ?? '';
     this.forbidCursorLine = !!props.forbidCursorLine;
     this.lang = ELang.Zh;
-
+    i18n.changeLanguage('cn');
```

Comment:
It need to be defined from props. And its default language need to be defined as `En`.